src/attachments/downloader.js

```diff
@@ -115,10 +121,17 @@ function copyFile(from, to) {
   });
 }
 
-function download(url, path) {
+export function download(url, path, headers) {
```

Comment:
If the `download` function was copy and pasted in both `downloadAttachments` and here, we should put it into its own ("internals") file and import it from there.

Reply:
Yes. We should refactor all http requests. But in the interest of time we needed a quick solution for the http-headers.